packages/react-component-library/src/components/TopLevelNavigation/SidebarE/SidebarUser.tsx

```diff
@@ -61,12 +63,16 @@ const StyledSidebarUserText = styled.div`
     justify-content: center;
     flex-direction: column;
 
-    span:first-of-type {
+    a:hover {
```

Comment:
Now that we know more about the patterns we want to adopt, could we consider refactoring this to use partials? Perhaps a separate ticket?

Reply:
Yeah good shout - i'll break things out into a partials directory.